test/create_and_test.sh

```diff
@@ -10,11 +10,11 @@ sudo docker exec -it experimenter-test bash -c "pip3 install coverage"
 sudo docker exec -it experimenter-test bash -c "pip3 install codecov"
 sudo docker exec -it experimenter-test bash -c "export CODECOV_TOKEN='66edf814-d6e1-40ae-b98a-dfea63a7e197'"
 sudo docker exec -it experimenter-test bash -c "coverage run run_tests.py"
```

Comment:
`sudo docker exec -it experimenter-test bash -c "coverage run run_tests.py"` needs to immediately preceed`val=$?`.